packages/components/src/product-icon/config.ts

```diff
@@ -132,7 +132,9 @@ export type SupportedSlugs =
 	| 'jetpack_backup_addon_storage_1tb_monthly'
 	| 'jetpack_backup_addon_storage_10gb_yearly'
 	| 'jetpack_backup_addon_storage_100gb_yearly'
-	| 'jetpack_backup_addon_storage_1tb_yearly';
+	| 'jetpack_backup_addon_storage_1tb_yearly'
+	| 'jetpack_backup_addon_storage_3tb_yearly'
```

Comment:
It would be unwise for us to ignore monthly terms, so thanks for pointing out their absence here. I pushed an update to add their slugs, and to put the new 3TB and 5TB plan slugs into the "backup addons" icon map.